build.gradle

```diff
@@ -6,20 +6,25 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.2.1"
-        classpath 'com.vanniktech:gradle-maven-publish-plugin:0.16.0'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
+plugins {
+  id 'signing'
+  id 'maven-publish'
```

Comment:
is this needed here? even if we're only publishing/signing the splunk-otel-java module?

Reply:
probably not